nncf/openvino/quantization/quantize_model.py

```diff
@@ -91,6 +93,57 @@ def dump_parameters(model: ov.Model, parameters: Dict, path: Optional[List] = No
         nncf_logger.debug(f"Unable to dump optimization parameters due to error: {e}")
 
 
+@tracked_function(NNCF_OV_CATEGORY, [CompressionStartedWithQuantizeApi(), "target_device", "preset"])
+def native_quantize_if_op_impl(
+    model: ov.Model,
+    calibration_dataset: Dataset,
+    preset: QuantizationPreset = QuantizationPreset.PERFORMANCE,
+    target_device: TargetDevice = TargetDevice.ANY,
+    subset_size: int = 300,
+    fast_bias_correction: bool = True,
+    model_type: Optional[ModelType] = None,
+    ignored_scope: Optional[IgnoredScope] = None,
+    advanced_parameters: Optional[AdvancedQuantizationParameters] = None,
+) -> ov.Model:
+    """
+    Implementation of the `quantize()` method for the OpenVINO backend via the OpenVINO Runtime API.
+    """
+    if not fast_bias_correction:
+        raise NotImplementedError(
+            "The BiasCorrection algorithm is not supported for OpenVINO models with If operation."
+        )
+    quantization_algorithm = PostTrainingQuantization(
+        preset=preset,
+        target_device=target_device,
+        subset_size=subset_size,
+        fast_bias_correction=fast_bias_correction,
+        model_type=model_type,
+        ignored_scope=ignored_scope,
+        advanced_parameters=advanced_parameters,
+    )
+
+    graph = GraphConverter.create_nncf_graph(model)
+
+    quantized_model = apply_algorithm_if_bodies(quantization_algorithm, model, graph, calibration_dataset, subset_size)
```

Comment:
By the way, why don't we want to call this method inside the `native_quantize_impl()` method? It would allow us to reduce the number of lines of code and I think it would be easier to maintain.
@alexsu52 @KodiaqQ

Reply:
Updated the `native_quantize_if_op_impl`